recipes/cylc/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "cylc" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  # Skip as Cylc requires Linux
+  skip: True  # [py<37 or not linux]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - 'configurable-http-proxy >=4.1'
+    - cylc-flow ==8.0a1
+    - cylc-uiserver ==0.1
+    - cylc-ui ==0.1
```

Comment:
Are these pins needed to be so tight?

Reply:
In theory yes.

We want this metapackage to provide everything users need to run a specific version of "Cylc". In the past, there was only `cylc`, which is now `cylc-flow`. We decided to match the version of the old package, so the metapackage version matches the `cylc-flow` release, so users are not confused.

This version of the Cylc metapackage should include

- `cylc-flow` with the same version
- `cylc-uiserver-0.1`, which handles GraphQL backend queries, and communication with `cylc-flow`
- `cylc-ui-0.1`, the JS application that initiates the calls

If users install everything as pinned above, it should work. If they install `cylc-ui ==0.2`, our next release of the UI, there is no guarantee it will work with the other dependencies. In fact, we know it shouldn't work as 0.2 has WebSockets, that is not implemented in `cylc-uiserver ==0.1`.

So for our next Cylc release 8.0a2 we will bump these pinned dependencies accordingly, so that users can `conda install cylc` and have the correct versions in their environments.

Hope that makes sense. This is our first release to Conda, as you can tell from our 0.1 versions, so happy to change it if you have suggestions for other ways to achieve something similar, or if this strict version requirement is an issue.

Thanks!

Reply:
It is up to you as the maintainer.  You will have to updated these pins manually at every release. But if you are OK with that...

Reply:
You can also always change this in the feedstock

Reply:
Thanks @scopatz 

For now we will probably keep these strict dependency requirements.

But once we have the initial 1.0.x release of each component, we will probably update it to something like `1.0.*` or `1.*`/

Thanks!
ps: appveyor unhappy again, but I guess it is again some glitch, and not related to this PR